packages/veritone-react-common/src/components/FormBuilder/FormItems/Text.js

```diff
@@ -11,6 +11,8 @@ export default function TextInput({ label, required, name, onChange, instruction
     [name, onChange],
   );
 
+  const styles = useStyles({});
```

Comment:
Remove if not used

Reply:
Actually we are using it, so we can not remove it.